README.md

```diff
@@ -41,26 +43,31 @@ Tracked Image
 
 ## Getting Started
 
-The fastest way to get started with DeepCell is to run the docker image:
+The fastest way to get started with `deepcell-tf` is to run one of our existing images from Docker Hub. To do so, first install [`nvidia-docker`](https://github.com/NVIDIA/nvidia-docker). Once `nvidia-docker` is installed, run the following command:
 
 ```bash
-nvidia-docker run -it --rm -p 8888:8888 vanvalenlab/deepcell-tf:0.4.0-gpu
+# Start a GPU enabled container on one GPUs
+docker run --gpus '"device=0"' -it --rm \
+	-p 8888:8888 \
+    -v $PWD/scripts:/notebooks \
+    -v $PWD/data:/data \
+    vanvalenlab/deepcell-tf:0.4.0-gpu
 ```
 
-This will start a Jupyter session, with several example notebooks detailing various training methods:
+This will spin up a docker container with `deepcell-tf` installed and start a jupyter session using the default port 8888. This command also connects a data folder ($PWD/data) and a scripts folder ($PWD/scripts) to the docker container so it can access your data and juyter notebooks stored on the host workstation. The default port can be changed to any non-reserved port by updating `-p 8888:8888` to, e.g., `-p 81:8888`. If you run across any errors getting started, you should either refer to the `deepcell-tf` for developers section or raise an issue on GitHub.
```

Comment:
I think the blurb "so it can access your data and notebooks" does a lot of that explanation already. Maybe just a sentence like "any saved data or models must be saved in the mounted directory to be accessible outside of the container" would suffice?

 not explain clearly enough that the container will not be able to access without it?

Additionally, we should change the word "connects" to "mounts" to use more google-able words.

Reply:
Done